simulateBankAccount/bankAccount.js

```diff
@@ -49,19 +58,25 @@ function createAccount(name, balance, maxWithdrawalLimit, minWithdrawalLimit, ma
         minWithdrawalLimit: minWithdrawalLimit
 
     }
+    //*************** 
+    app.post('/clients', (req,res) =>{
```

Comment:
I think your endpoints should rather live outside of the functions they are currently in, and rather call the relevant functions within them